.rubocop.yml

```diff
@@ -3,7 +3,7 @@ inherit_from: .rubocop_todo.yml
 require: rubocop-rspec
 
 AllCops:
-  TargetRubyVersion: 3.0
+  TargetRubyVersion: 2.7
```

Comment:
oh phooey.  Because this is used in both.

Reply:
yep!

Reply:
presumably it'll still work in ruby 3 apps with the target version of 2.7?

i just pulled `main` after this merge, and was able to successfully run the test suite using ruby 3 even after this fix.

i guess we'll see whether the consumers complain when dep updates come around!

Reply:
Yeppers.